ddl/ddl_tiflash_test.go

```diff
@@ -50,14 +51,12 @@ type tiflashDDLTestSuite struct {
 	cluster *unistore.Cluster
 }
 
-var _ = SerialSuites(&tiflashDDLTestSuite{})
-
 const (
 	RoundToBeAvailable               = 2
 	RoundToBeAvailablePartitionTable = 3
 )
 
-func (s *tiflashDDLTestSuite) SetUpSuite(c *C) {
+func SetUpHelper(s *tiflashDDLTestSuite) error {
```

Comment:
We don't write test in this style. Please checkout other test file in testify.

Reply:
I don't quite follow... I am trying to follow examples like `TestAnalyzeIndexExtractTopN` before migrate the test to testify. `tiflashDDLTestSuite` is something from gocheck, it only act as a combination of context now.